Resources/views/PlatformUI/shell.html.twig

```diff
@@ -15,9 +15,15 @@
         </script>
         <script type="text/javascript" src="//cdn.crowdin.com/jipt/jipt.js"></script>
     {% endif %}
-{% stylesheets '$css.files;ez_platformui$' filter='cssrewrite' %}
-    <link rel="stylesheet" href="{{ asset_url }}"/>
-{% endstylesheets %}
+    {% if parameters.css.combine %}
+        {% stylesheets '$css.files;ez_platformui$' filter='cssrewrite' combine=true %}
+            <link rel="stylesheet" href="{{ asset_url }}"/>
+        {% endstylesheets %}
+    {% else %}
+        {% stylesheets '$css.files;ez_platformui$' filter='cssrewrite' combine=false %}
```

Comment:
self nitpick, I realize this should not be set to false but just use system default. otherwise it will be false also for prod.